templates/ovncontroller/bin/stop-ovsdb-server.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+#
+# Copyright 2024 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+ovs_folder=/var/lib/openvswitch
+
+# ovs_vswitchd container needs to terminate before ovsdb-server because it need
+# access to the db on the preStop script. This semaphore ensures it is not torn
+# down.
+while [ ! -f $ovs_folder/finish_ovsdb_server ]
+do
+  sleep 0.1
```

Comment:
This seems very aggressive. Maybe 0.5 or 1?..

Reply:
done!